tests/views_wpunit/Tribe/Events/Views/V2/Template_BootstrapTest.php

```diff
@@ -245,6 +247,62 @@ public function should_not_filter_a_404_template() {
 		$this->assertEquals( 'foo/bar.php', $bootstrap->filter_template_include( 'foo/bar.php' ) );
 	}
 
+
+	/**
+	 * It should return the event template path when unfiltered.
+	 *
+	 * @test
+	 */
+	public function it_should_return_the_event_template_path_when_unfiltered() {
+		$default_events = '/var/www/html/wp-content/plugins/the-events-calendar/src/views/v2/default-template.php';
+		// Run our "faked" events query.
+		$this->setup_event_query();
+
+		$template = tribe( Template_Bootstrap::class )->filter_template_include( 'foo-bar' );
+		$this->assertEquals( $template, $default_events, "Template path should not be 'foo-bar' on the `embed_template` hook when unfiltered." );
```

Comment:
Please use the arguments correctly: the first one is `$expected` so it should be `$default_events`; the second one is `$actual` so it should be `$template`.

Reply:
This flip of arguments should be done here and elsewhere.

Reply:
```suggestion
		$this->assertEquals( $default_events, $template, 'Template path should not be 'foo-bar' on the `embed_template` hook when unfiltered.' );
```